### PR TITLE
update docker docs regarding volume services

### PR DESCRIPTION
**Pull request by jeffpak**
Sorry for the whitespace diffs! We made a couple additions, which include the Docker Volume Support section and an addendum to the Docker Security Concerns in a Multi-Tenant Environment.

[#139759481](https://www.pivotaltracker.com/story/show/139759481)

Signed-off-by: Jeff Pak <jeffrey.pak@emc.com>

### CODE REVIEW

**Comment by kdolor-pivotal** [comment]
Thanks @jeffpak !

File: docker.html.md.erb [hunk 1]
```diff
@@ -82,25 +82,29 @@ $ cf disable-feature-flag diego_docker
...
- <pre class='terminal'>$ cf push my-app -o MY-PRIVATE-REGISTRY.DOMAIN:5000/image/name:v2</pre> 
+ <pre class='terminal'>$ cf push my-app -o MY-PRIVATE-REGISTRY.DOMAIN:5000/image/name:v2</pre>
 
- Replace `MY-PRIVATE-REGISTRY.DOMAIN` with your domain name, which resolves to the private registry. 
+ Replace `MY-PRIVATE-REGISTRY.DOMAIN` with your domain name, which resolves to the private registry.
 In this example, 5000 is the port on which your private registry serves traffic. `image/name` is the name of the Docker image repository. `v2` is a tag in that repository that refers to a specific image.
+
+###<a id='volume'></a> Docker Volume Support
+
+Diego now supports Docker volumes. More information can be found [here](https://docs.cloudfoundry.org/devguide/services/using-vol-services.html) on how to enable volume support, and the limitations of NFS volumes within the [NFS Bosh Volume Release](https://github.com/cloudfoundry-incubator/nfs-volume-release#special-notes-for-docker-image-based-apps).
```
Comment (emalm) [inline docker.html.md.erb]: @julian-hj @jeffpak The statement "Diego now supports Docker volumes" is confusing, and makes me think that CF does something magical when it sees a `VOLUME` directive in Docker image metadata when it may instead mean that the Diego cells are compatible with Docker volume plugins. It's now present at https://github.com/cloudfoundry/docs-dev-guide/blob/880131089e0c12a7e0b2b052c875f2c0ae37590e/deploy-apps/push-docker.html.md.erb#L144, but I think we should reword it to be clearer. /cc @seviet
